lib/src/server/hooks.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
+import 'package:github/src/common/model/changes.dart';
```

Comment:
nit: 

```suggestion
import '../common/model/changes.dart';
```

https://dart.dev/tools/linter-rules/prefer_relative_imports

Reply:
Ah okay. I had this same question for within flutter because we do not use this rule.